airbyte-scheduler/src/main/java/io/airbyte/scheduler/SchedulerApp.java

```diff
@@ -96,10 +99,11 @@ public void start() {
           jobRetrier.run();
           jobScheduler.run();
           jobSubmitter.run();
+          jobCleaner.run();
```

Comment:
I am afraid this is going to be too frequent and will generate too much IO

Reply:
good point. i guess nothing prevent us from just throwing this in a separate `scheduleWithFixedDelay`

Reply:
Switching to every two hours for cleaning.